kernel/sched.c

```diff
@@ -1033,15 +1034,17 @@ struct k_thread *z_priq_rb_best(struct _priq_rb *pq)
 # endif
 #endif
 
-ALWAYS_INLINE void z_priq_mq_add(struct _priq_mq *pq, struct k_thread *thread)
+static ALWAYS_INLINE void z_priq_mq_add(struct _priq_mq *pq,
+					struct k_thread *thread)
```

Comment:
I did not notice this before, but a forward declaration is going to be needed for this routine (and likely others)--d'oh! This routine is used (via a macro) at an earlier location in the file.